srtcore/srt_compat.c

```diff
@@ -22,7 +22,8 @@ written by
 #include <stdio.h>
 #include <errno.h>
 #if !defined(_WIN32) \
-   && !defined(__MACH__)
+   && !defined(__MACH__) \
+   && !(defined(__unix__) && defined(BSD))
```

Comment:
Well, isn't that kinda simpler - that is, Mac and Linux only?

Reply:
what about solaris or other operating systems that do have features.h. There are more thing in heaven and earth Horatio than fit into your philosphy.

Reply:
I commited a change to simplify this to check for __unix__ and not BSD. BSD seems to be defined on all the BSD derived OSes including OSX. I am not sure however whether that is the case for iOS, Watch, and AppleTV OSes tho. But I assume it is. Like for instance Android != linux, but does defined __unix__ and __linux__ .

Reply:
Important thing for it is not that it should be enabled on every system that features `features.h`, but to be enabled on every system where we want something specific from its `features.h`. We might have some "sensible default portable" method for systems that this project wasn't tested against, but this should simply NOT try to include this file at all, even if it exists.